src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {' '}
+    <div className="CommentList">
+      {comments.map(comment => {
+        return <CommentInfo comment={comment} key={comment.id} />;
+      })}
+    </div>
+  </>
```

Comment:
Looks weird - you use the fragment just to add whitespace before your `div`.
Do you really need this?